test/normalize.spec.js

```diff
@@ -703,6 +744,231 @@ describe('embedded', () => {
 
     expect(result).to.deep.equal(output);
   });
+
+  it('embedded list without standalone link ignores link array', () => {
```

Comment:
Not sure I understand how the link array is ignored in this test. Are you saying there could be different data in `_links.questions` and it should still yield the same output? The current test doesn't really prove that the link array is ignored in my opinion.

Reply:
Yeah, the title seems a bit misleading. I think a better wording would be "keep list embedded if data provided in links is not a standalone link".

What's happening here is the following: Before this PR, we would check if `_links.question`s exists (which it does) and use it as a standalone link (=[accessing it's href property](https://github.com/carlobeltrame/hal-json-normalizer/pull/64/files#diff-7f7c0e0d6a9d315240bed1f927d895f579dd0ff44188355588383be7ce26574dL104)). Which leads to an error, because `_links.questions` is an array and `_links.questions.href` doesn't exist.

Now I'm checking first, if `_links.questions` is really a standalone link (`isSingleLink`). And this is the test for it.